misc/attrib.pm

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/perl
+#
+=head1  NAME
+
+    attrib.pm - two subs to handle ARQ and RRQ message attributes for HPT perl
+    by Stas Mishchenkov 2:460/58.
+
+=head1 SYNOPSIS
+
+   use attrib;
+   
+   sub filter()
+   {
+      is_rrq( $attr );
+   }
+   
+   sub route()
+   {
+      is_arq( $attr, $route );
+   }
+   
+   $attr - message attributes as defined by HPT perl.
+   $route - message routing as defined by HPT perl.
+
+=head1 DESCRIPTION
+
+   This subs to handle ARQ and RRQ message attributes for HPT perl designed
+   according to FTS-0001.016.
+   Insert into HPT configuration file:
+      hptperlfile /home/fido/perl/filter.pl
+
+   Put attrib.pm somewhere in the @INC path. It's strongly recommended for Windows
+   users to put it in the same directory with filter.pl.
+   place to filter.pl something like this:
```

Comment:
Instead of 
> place to filter.pl something like this 

it is better to say "You may use the subroutines inside filter.pl this way:". Please pay attention to the capital letter at the beginning of the sentence.